src/tests/Common/Coreclr.TestWrapper/CoreclrTestWrapperLib.cs

```diff
@@ -343,6 +578,32 @@ public int RunTest(string executable, string outputFile, string errorFile, strin
                         exitCode = process.ExitCode;
                         MobileAppHandler.CheckExitCode(exitCode, testBinaryBase, category, outputWriter);
                         Task.WaitAll(copyOutput, copyError);
+
+                        if (!OperatingSystem.IsWindows())
+                        {
+                            // crashreport is only for non-windows.
+                            if (exitCode != 0)
+                            {
+                                // Search for dump, if created.
+                                if (Directory.Exists(crashDumpFolder))
+                                {
+                                    outputWriter.WriteLine($"Test failed. Trying to see if dump file was created in {crashDumpFolder} since {startTime}");
+                                    DirectoryInfo crashDumpFolderInfo = new DirectoryInfo(crashDumpFolder);
+                                    var dmpFilesInfo = crashDumpFolderInfo.GetFiles("*.crashreport.json").OrderByDescending(f => f.CreationTime);
```

Comment:
nit: we tend to use var only on cases where we know the type.

Reply:
not sure what you mean - Here, it is `IOrderedEnumerable<FileInfo>` and I just thought `var` would be shorter to write.